platforms/documentation/docs/src/samples/credentials-handling/pass-credentials-to-external-tool-via-stdin/groovy/build.gradle

```diff
@@ -2,7 +2,7 @@ def login = tasks.register('login', Exec) {
     def loginProvider = providers.credentials(PasswordCredentials, 'login')
     inputs.property('credentials', loginProvider)
 
-    commandLine = ['sh', 'login.sh']
+    commandLine(['sh', 'login.sh'])
```

Comment:
❓ This looks like a step backwards, no?

Reply:
Maybe, not sure. :) The thing is, that command line is calculated from multiple properties, e.g. executable and args. So we somehow have to make sure that properties are in sync if `commandLine` is set, or we have to document that executable and args won't reflect `commandLine` if it's set directly.

So I think the best option is to just not allow set `=` commandlIne with `=`.

Reply:
I see, so `commandLine()` is a helper method where the first parameter is the executable, the rest are arguments... 🤔 But then the code in `build-logic-settings` (at the top of the PR's diff) won't work anyumore, no?

Reply:
Good point, yes it will fail, fixed